code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -136,10 +136,31 @@
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/plasma/can_attach(obj/mecha/working/M)
 	if(..()) //combat mech
-		return 1
+		return TRUE
 	else if(M.equipment.len < M.max_equip && istype(M))
-		return 1
-	return 0
+		return TRUE
+	return FALSE
+
+/obj/item/mecha_parts/mecha_equipment/weapon/energy/pka_carbine
+	equip_cooldown = 30
+	name = "Proto-Kinetic carbine"
+	desc = "An high pressure kinetic accelerator which shoots a burst of kinetic force in rapid succession."

```

Comment:
Change this to "A high pressure kinetic accelerator which shoots bursts of kinetic force in rapid succession."

Reply:
what